daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -1341,6 +1341,22 @@ message Update {
     Expr arg = 5;
   }
 
+  // Dynamic Exercise Update
+  // *Available in versions >= 1.dev*
+  message DynamicExercise {
```

Comment:
> I would prefer to have an explicit UpdateDynamicExercise expression in order to isolate better production code form PoC code. I do not think it is a big peace of work taking into account there is already a Sbuiltin for that.

done now

Reply:
Great, I was going to suggest the same. This looks really good now :)